www/api/class/centreon_clapi.class.php

```diff
@@ -230,4 +235,12 @@ public function authorize($action, $user, $isInternal = false)
 
         return $user->hasAccessRestApiConfiguration();
     }
+
+    /**
+     * Removes carriage returns from $item if string
+     * @param $item variable to check
+     */
+    private function clearCarriageReturns(&$item) {
```

Comment:
The name of the method is confusing because it does not only remove carriage returns (\t)